specification/msi/resource-manager/Microsoft.ManagedIdentity/2018-11-30/ManagedIdentity.json

```diff
@@ -535,7 +535,10 @@
       "in": "query",
       "description": "Version of API to invoke.",
       "required": true,
-      "type": "string"
+      "type": "string",
+      "enum": [
+        "2018-11-30"
+      ]
```

Comment:
I am not sure if the enum for the api-version parameter is required.

@veronicagg - Is this allowed?

If not then please remove it.